Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -434,6 +434,7 @@
         "MSDeploygeneratedpackageareonlysupportedforWindowsplatform": "MSDeploy generated packages are only supported for Windows platform.",
         "UnsupportedinstalledversionfoundforMSDeployversionshouldbeatleast3orabove": "Unsupported installed version: %s found for MSDeploy. version should be at least 3 or above",
         "UnabletofindthelocationofMSDeployfromregistryonmachineError": "Unable to find the location of MS Deploy from registry on machine (Error : %s)",
+        "UnabletofindthelocationofDotNetFrameworkfromregistryonmachineError": "Unable to find the location of Dot Net Framework from registry on machine (Error : %s)",
```

Comment:
You can define the resource in module.json within the common module, instead of having to define them in individual tasks. Refer module.json in common\java-common as example.

Reply:
Resources in modules.json files are not picked up during build. This is a known issue with Xplat tasks.
We need to add the entry in module.json and also copy the strings to the tasks.